core_pb/src/messages/settings.rs

```diff
@@ -46,6 +46,19 @@ impl Default for PacbotSettings {
     }
 }
 
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, PartialOrd)]
+pub enum TargetPath {
```

Comment:
I don't love the name here, we will very soon have other things in the app related to how the target path is decided and that could get confusing. Can we name it ShouldDoTargetPath, with options Yes, No, and DoWhilePlayed? a long name is ok as this won't be used in many places